code/game/machinery/doors/blast_door.dm

```diff
@@ -123,15 +128,33 @@
 					usr << "<span class='warning'>You don't have enough sheets to repair this! You need at least [amt] sheets.</span>"
 
 
+		else if(src.density)
+			var/obj/item/weapon/W = C
+			user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*2)
```

Comment:
Resulting in more strikes, which I don't agree with. However since you insist, I'll do that then.